code/game/objects/structures/crates_lockers/closets/secure/misc.dm

```diff
@@ -65,3 +65,39 @@
 	new /obj/item/weapon/storage/box/metalfoam(src)
 	for(var/i in 1 to 3)
 		new /obj/item/weapon/rcd_ammo/large(src)
+
+		/obj/structure/closet/secure_closet/iaa
+	name = "iaa's Closet"
+	desc = "Internal Affairs Closet"
+	req_access = list(ACCESS_CENT_SPECOPS)
+	icon_state = "iaa"
+
+/obj/structure/closet/secure_closet/iaa/PopulateContents()
+	..()
+	new /obj/item/weapon/storage/belt/holster/full(src)
```

Comment:
It was discussed... Last night I think? Maybe the day before? And ecolli has been convinced that the IAA does not need a gun. Remove the thing.